extensions/amp-analytics/0.1/variables.js

```diff
@@ -27,6 +28,13 @@ const TAG = 'amp-analytics/variables';
 /** @const {RegExp} */
 const VARIABLE_ARGS_REGEXP = /^(?:([^\s]*)(\([^)]*\))|[^]+)$/;
 
+const EXTERNAL_CONSENT_POLICY_STATE_STRING = {
```

Comment:
Are these values documented? 🤔

Reply:
not yet, I'll need to when I document the feature in the following PR.